tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_api.c

```diff
@@ -118,3 +162,55 @@ void test_k_aligned_alloc(void)
 		"%p not 16-byte-aligned", r);
 	k_free(r);
 }
+
+/**
+ * @brief Validate allocation and free from system heap memory pool
+ *
+ * @ingroup kernel_heap_tests
+ *
```

Comment:
please add the @details to descript the high level testing purpose or concept, thanks!

Reply:
Hi, enjia.  Details have been added. Thanks.